crates/cxx-qt-gen/src/parser/method.rs

```diff
@@ -34,22 +35,12 @@ impl ParsedQInvokableSpecifiers {
 
 /// Describes a single method (which could be a Q_INVOKABLE) for a struct
 pub struct ParsedMethod {
-    /// The original [syn::ImplItemFn] of the invokable
-    pub method: ForeignItemFn,
-    /// The type of the self argument
-    pub qobject_ident: Ident,
-    /// Whether this invokable is mutable
-    pub mutable: bool,
-    /// Whether the method is safe to call.
-    pub safe: bool,
-    /// The parameters of the invokable
-    pub parameters: Vec<ParsedFunctionParameter>,
+    /// The common fields which are available on all callable types
+    pub method_fields: MethodFields,
     /// Any specifiers that declared on the invokable
     pub specifiers: HashSet<ParsedQInvokableSpecifiers>,
     /// Whether the method is qinvokable
     pub is_qinvokable: bool,
-    /// The rust and cxx name of the function
-    pub name: Name,
 }
```

Comment:
shouldn't there be `doc` here, as a separate item, like in ParsedSignal / ParsedInheritedMethod ?

Reply:
I think I discussed this with @LeonMatthesKDAB , and since you have to implement your methods outside the bridge, the docs used are the ones from the implementation (which are also required by clippy), so any docs given here are ignored. Might need to confirm though.

Reply:
aha, makes sense, maybe add a comment here stating this as it's an oddity otherwise when you compare the structs :-)

Reply:
yep will do